File: cpp/program/Peripheral.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class Peripheral{
protected:
    string merk;
    string jenis;
public:
    Peripheral(){}

    Peripheral(string merk, string jenis){
        this->merk = merk;
        this->jenis = jenis;
    }

    void setmerk(string merk){
        this->merk = merk;
    }
    void setjenis(string jenis){
        this->jenis = jenis;
    }

    string getmerk(){
        return this->merk;
    }
    string getjenis(){
        return this->jenis;
    }

    ~Peripheral(){}
};

File: python/program/Peripheral.py
class Peripheral:
    def __init__(self, merk='', jenis=''):
        self.merk = merk
        self.jenis = jenis

    def set_merk(self, merk):
        self.merk = merk

    def set_jenis(self, jenis):
        self.jenis = jenis

    def get_merk(self):
        return self.merk

    def get_jenis(self):
        return self.jenis